math/calculator.cpp

```diff
@@ -0,0 +1,168 @@
+/**
+ * Copyright 2020 @author David Leal
+ * @file
+ *
+ * Basic calculator: sum, subtract, multiply, and divide.
+ */
+
+#include <iostream>
+#include <conio.h>
+#include <cstdlib>
+
+struct Node {
+    float datatype;
+};
+
+// Prototypes
+void requestData();
+void pause();
+
+Node* new_node = new Node();
+
+int opt = 0;
+float number;
+
+int main() {
+    requestData();
+
+    _getch();
+    return 0;
+}
+
+void pause() {
+    std::cout << "Press any key to continue . . . ";
+    _getch();
```

Comment:
`_getch()` is provided by `conio.h` and not available by GCC or clang and not part of ISO C++ standard